tests/transports-governance/conftest.py

```
@@ -0,0 +1,668 @@
+        **kwargs,
+    ) -> requests.Response:
+        """Make chat completion request"""
+        data = {"model": model, "messages": messages, **kwargs}
+
+        session_headers = self.session.headers.copy()
+        if headers:
+            session_headers.update(headers)
+
+        url = f"{COMPLETION_API_BASE}/chat/completions"
+        try:
+            response = requests.post(url, json=data, headers=session_headers)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)